libs/application-generic/src/schemas/control/email-control.schema.ts

```diff
@@ -11,6 +11,7 @@ export const emailControlZodSchema = z
     editorType: z.enum(['block', 'html']).optional().default('block'),
     subject: z.string().min(1),
     disableOutputSanitization: z.boolean().optional(),
+    layoutId: z.string().nullable().optional(),
```

Comment:
`layoutId` could be `null` which means that the user don't want to use any, `undefined` will mean that the default layout should be used